website/docs/d/service.html.markdown

```diff
@@ -19,6 +19,14 @@ data "kubernetes_service" "example" {
     name = "terraform-example"
   }
 }
+
+resource "aws_route53_record" "example" {
+  zone_id = "${data.aws_route53_zone.k8.zone_id}"
+  name    = "example"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["${data.kubernetes_service.nginx.load_balancer_ingress.0.hostname}"]
```

Comment:
I think this should be `example` rather than `nginx`?

Reply:
Good catch, fixed.